ArduPlane/servos.cpp

```diff
@@ -912,7 +914,7 @@ void Plane::set_servos(void)
     }
 
     float override_pct = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
-    if (g2.ice_control.throttle_override(override_pct)) {
+    if (g2.ice_control.throttle_override(override_pct, base_throttle)) {
```

Comment:
Could use `get_throttle_input(true)` here to always pass manual throttle independent of mode

Reply:
we don't want it in all modes - imagine we are in AUTO or LOITER or QRTL - we don't want to start passing through throttle when the mode doesn't normally pass thru throttle